src/careamics/models/lvae/lvae.py

```diff
@@ -4,74 +4,71 @@
 The current implementation is based on "Interpretable Unsupervised Diversity Denoising and Artefact Removal, Prakash et al."
 """
 
-from typing import Dict, Iterable, List, Tuple, Union
+from collections.abc import Iterable
+from typing import Dict, List, Tuple
 
-import ml_collections
+import ml_collections  # TODO: refactor this out
```

Comment:
I don't see it used except in the declaration of `config: ml_collections` and then this `config` is not used (as far as I can tell). So kick it out as we discussed to allow the tests from passing (hopefully)!